fala/apps/adviser/models.py

```diff
@@ -10,3 +14,145 @@ class SatisfactionFeedback(models.Model):
 
     def __str__(self):
         return f"Feedback {self.id}"
+
+
+class EnglandOrWalesState(object):
```

Comment:
With classes that we want to re-use we can do the following:

```
fala/
├── common/
│   ├── __init__.py
│   ├── states.py  # New file for reusable classes like EnglandOrWalesState
├── app/adviser/
│   ├── views.py
├── app2/single-page/
│   ├── views.py
```

common will be a directory that we will use to access anything we need across all of our apps.

Reply:
created a ticket for this refactor:
https://dsdmoj.atlassian.net/browse/EL-1981